mne/event.py

```diff
@@ -444,6 +444,13 @@ def _find_events(data, first_samp, verbose=None, output='onset',
         data = np.abs(data)  # make sure trig channel is positive
 
     events = _find_stim_steps(data, first_samp, pad_stop=0, merge=merge)
+    if data[0, 0] != 0:
+        if initial_event:
+            events = np.insert(events, 0, [0, 0, data[0, 0]], axis=0)
+        else:
+            warn('Trigger channel has a non-zero initial value (consider using'
+                 ' initial_event=True to detect this event)')
```

Comment:
Might as well give the initial value.

If we're going to warn here, we should probably support `initial_value='ignore'` or something similar to squash this message, and say that's an option in the warning, too. (The user could do `verbose='error'` but it's not as clean because it suppresses any other potentially useful warnings/infos.)

Reply:
+1 for giving the actual value. We don't have to warn, if it is easier I can change it log an info message.

Reply:
`info` sounds good to me